packages/README.md

```diff
@@ -0,0 +1,41 @@
+Packages
+================
+
+A package allows the creation of modular code that cen easily be shared. For example, packages
```

Comment:
at line 4, `cen` should be `can`, small typo 👍

Reply:
typo corrected